readthedocs/ssh/forms.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from __future__ import division, print_function, unicode_literals
+
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+from .keys import is_valid_private_key
+
+
+class SSHKeyFileUploadForm(forms.Form):
+
+    private_key = forms.FileField(
+        help_text=_('Choose the private key file from your computer that want to upload and use to clone this project'),  # noqa
```

Comment:
A couple things on copy, to improve clarity:

* This key might be used to install via pip or clone a repository, so perhaps we just leave that out of this comment
* `project` is a Read the Docs term, users `clone` a `repository`
* `from your computer` is not needed and might be distracting

Perhaps something closer to:

`Upload an additional private key file to use while cloning or installing dependencies`

Or something similar, of course.

Reply:
You suggestion makes sense to me :)